src/child/index.js

```diff
@@ -20,6 +21,15 @@ require('script!../../node_modules/BitFlux/dist/bitflux.js');
 
 const store = configureStore();
 
+fin.desktop.main(() => {
+    fin.desktop.Window.getCurrent().addEventListener('bounds-changed', (e) => {
```

Comment:
The `previousExpandedDimensions` would also have to be saved when maximising so we can support the use case:
```
resize normal view -> maximise -> compact -> full view (to maximised) -> restore (back to resized normal view)
```
This is fine if the maximise is performed via the `Toolbar` since we can get the current window size before we perform the actual window resize. The problem is when the maximise is performed via the OS/windows taskbar - in this case the window resize operation has already happened before we get the chance to save the original window dimensions.